game_eggs/steamcmd_servers/squad/README.md

```diff
@@ -1,9 +1,14 @@
+
 # Squad
 
-Squad is an online multiplayer first-person shooter that aims to capture combat realism through communication and teamplay.
+## Steam Description
+Squad is a tactical FPS that provides authentic combat experiences through teamwork, communication, and realistic combat. It bridges the gap between arcade shooter and military realism with 100-player battles, combined-arms warfare, and base building.
 
-## Server Ports
+## Versions
+* Vanilla - The standard Squad server optimized for Linux environments. Offers maximum stability and performance.
```

Comment:
You removed the vanilla egg >.>

Reply:
I added that description, I didn't expect @fakhry7050 will delete it 😅

Reply:
Still needs fixed 🤷🏼

Reply:
Fixed 🎉